SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md

```diff
@@ -51,6 +51,8 @@ You can find out more about the Console in the [Glossary]({{urlRoot}}/content/gl
 
 **Note:** You must close the Unreal Editor before building your workers. If the Editor is open when you try to build your workers the command will fail.
 
+**Note:** Ensure that you have enabled Spatial networking for your game before building worker assemblies. DefaultGame.ini configuration file located in `<ProjectRoot>\Game\Config` directory must contain a line `bSpatialNetworking=True` under `[/Script/EngineSettings.GeneralProjectSettings]` section.
```

Comment:
Would be good if someone from Unreal GDK team could confirm whether this is enough to enable Spatial networking.

Reply:
Yes - this is all you need. It actually defaults to true, so if it's present in the config file, it needs to be set to true, otherwise it doesn't need to be detailed.

Reply:
Good to know, thanks @m-samiec! Do you think this should be paraphrased to say that you don't have to worry about it if you don't have `bSpatialNetworking` specified? Or is it fine?

Reply:
It's probably good to clarify. Something like -

** Note: ** Spatial networking is default enabled for Unreal GDK projects. If the `bSpatialNetworking` option is visible in your `DefaultGame.ini`, make sure it's set to true.

Reply:
How about this?

**Note:** By default, Spatial networking is enabled for Unreal GDK projects. However, if the `bSpatialNetworking` option is visible in your `DefaultGame.ini` configuration file (located in `<ProjectRoot>\Game\Config` directory), ensure that it is set to `True` to enable Spatial networking for your cloud deployment.

Reply:
Looks good! Worth having a tech-writer cast their eyes over it - @ElleEss / @NotRobSheridan ?